sherpa-ncnn/csrc/recognizer.cc

```diff
@@ -110,6 +140,31 @@ class Recognizer::Impl {
     } else if (config.decoder_config.method == "modified_beam_search") {
       decoder_ = std::make_unique<ModifiedBeamSearchDecoder>(
           model_.get(), config.decoder_config.num_active_paths);
+      std::vector<int32_t> tmp;
+      /*each line in hotwords file is a string which is segmented by space*/
+      std::ifstream file(config_.hotwordsfile);
+      if (file) {
+        std::string lines;
+        std::string word;
+        while (std::getline(file, lines)) {
+          std::istringstream iss(lines);
+          std::cout << lines << std::endl;
+	      while(iss >> word){
+            if (sym_.contains(word)) {
```

Comment:
Could you indent the `if` inside `while` correctly?
(Hint: Please use spaces to replace tabs.)

Reply:
done